packages/apps/composer-app/src/components/Placeholder.tsx

```diff
@@ -0,0 +1,37 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import React, { useEffect, useState } from 'react';
+
+import { DXOS } from '@dxos/brand';
+import { Status, ThemeProvider } from '@dxos/react-ui';
+import { defaultTx, getSize, mx } from '@dxos/react-ui-theme';
+
+export const Placeholder = () => {
+  const [className, setClassName] = useState<string>();
+  useEffect(() => {
+    setTimeout(() => {
+      setClassName('transition duration-1000 rotate-180 scale-100 opacity-25 blur-none');
+    }, 100);
+  }, []);
+
+  return (
+    <ThemeProvider tx={defaultTx}>
+      <div className='flex flex-col bs-[100dvh] justify-center items-center'>
+        <Status indeterminate aria-label='Initializing' classNames='block is-full' />
+        <div className={mx('flex grow items-center scale-50 opacity-0 blur-xl', className)}>
+          <DXOS className={getSize(48)} />
```

Comment:
I don’t think this improves usability much, and it uses our 'X' logo which is already a signifier in UX for ‘dismiss’ or ‘close’.

I would much prefer a loading skeleton, for which there are plenty of studies that show usability improvements. Now that Deck has a stable design, we should seriously consider skeletons instead of trying to optimize an atomic signifier like this.

Reply:
See: https://www.nngroup.com/articles/skeleton-screens/

Reply:
Let's discuss.

Reply:
Noting for posterity from the NN article:

> Spinners are typically best used on a single module, like a video or a card which is on a dashboard. Skeleton screens (with the exception of frame-display ones) are better when the full screen is loading because the wireframe gives users a sense of what the page will look like and, thus, minimizes cognitive load.

> progress bars are strongly recommended for any page that takes longer that 10 seconds to load because they give users a sense of the state of the system and of how much longer they have to wait. Anything above 10 seconds requires an explicit estimation of duration.